### PR TITLE
add us.zoom.Zoom

### PR DESCRIPTION
**Pull request by nedrichards**
Adds Zoom Video Conferencing app https://zoom.us 

Needs device=all for video chat.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build us.zoom.Zoom

**Comment by nedrichards** [comment]
@TingPing for this one I've followed the practice from slack where there isn't an established appid rather than Skype or Spotify (which add .Client). I'm not sure which is better.

File: us.zoom.Zoom.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  <id>us.zoom.Zoom.desktop</id>
+  <name>Zoom</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>zoom.us</developer_name>
+  <summary>Video Conferencing, Web Conferencing, Webinars, Screen Sharing</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://zoom.us</url>
+  <description>
+    <p>Award winning Zoom brings video conferencing, online meetings and group messaging into one easy-to-use application.</p>
+    <p>One consistent enterprise experience for all use cases</p>
```
Comment (TingPing) [inline us.zoom.Zoom.appdata.xml]: Perhaps these should be an unordered list rather than one sentence paragraphs?

**Comment by TingPing** [comment]
> @TingPing for this one I've followed the practice from slack where there isn't an established appid rather than Skype or Spotify (which add .Client). I'm not sure which is better.

The last segment is largely arbitrary but Zoom seems more descriptive.

File: us.zoom.Zoom.appdata.xml (hunk 1 above)
Comment (nedrichards) [reply us.zoom.Zoom.appdata.xml]: Yeah, that's fair.

**Comment by nedrichards** [comment]
bot, build us.zoom.Zoom

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/us.zoom.Zoom

**Pull request closed without merging**